frontend/document/koptinterface.lua

```diff
@@ -340,7 +341,7 @@ end
 
 function KoptInterface:renderPage(doc, pageno, rect, zoom, rotation, gamma, render_mode, hinting)
     if doc.configurable.text_wrap == 1 then
-        return self:renderReflowedPage(doc, pageno, rect, zoom, rotation, render_mode, hinting)
+        return self:renderReflowedPage(doc, pageno, rect, zoom, rotation, render_mode)
```

Comment:
Here's the prototype:
```lua
function KoptInterface:renderReflowedPage(doc, pageno, rect, zoom, rotation, render_mode)
```
No `hinting` parameter.

Reply:
renderReflowedPage doesn't do anything with it at this time, although I'm not sure if that's necessarily an argument to remove it since maybe then you'd be touching the very same line again next week. :-)